src/app/public/modules/angular-tree/angular-tree-node.component.html

```diff
@@ -54,7 +50,11 @@
     <!-- https://github.com/500tech/angular-tree-component/blob/master/lib/components/tree-node-wrapper.component.ts -->
     <div
       class="node-content-wrapper"
+      role="treeitem"
       [allowDragoverStyling]="node.allowDragoverStyling()"
+      [attr.aria-current]="node.isActive"
+      [attr.aria-expanded]="node.isExpanded"
+      [attr.aria-selected]="ariaSelected()"
```

Comment:
☝️ The `treeitem` role should be set on the element that is interactive and focusable. https://www.w3.org/TR/wai-aria-practices/examples/treeview/treeview-2/treeview-2a.html